src/cmd/script.rs

```diff
@@ -15,20 +15,20 @@ use common_path::common_path_all;
 use std::collections::HashSet;
 
 /// Assemble the `script` subcommand.
-pub fn new<'a, 'b>() -> App<'a, 'b> {
-    SubCommand::with_name("script")
+pub fn new<'a>() -> Command<'a> {
+    Command::new("script")
         .about("Emit tool scripts for the package")
         .arg(
-            Arg::with_name("target")
-                .short("t")
+            Arg::new("target")
+                .short('t')
                 .long("target")
                 .help("Only include sources that match the given target")
                 .takes_value(true)
-                .multiple(true)
+                .multiple_values(true)
```

Comment:
Should `.multiple_occurrences(true)` also be added here, like for the `path` command? [According to the `clap` doc](https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.multiple):
> Setting `multiple(true)` for an [option](https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.takes_value) with no other details, allows multiple values **and** multiple occurrences [...]

The same question raises below for the arguments `define`, `vcom-arg`, and `vlog-arg`, where we had `multiple(true)`, and for `vlogan-bin` and `vhdlan-bin`, where we had `multiple(false)`.

Reply:
I have updated the change to the `multiple` option. As far as I can tell, the following should hold:
- `.multiple(true).number_of_values(1)` -> `.multiple_occurrences(true)`
- `.multiple(true)` -> `.multiple_values(true).multiple_occurrences(true)`
For some things, `multiple_occurrences` does not make much sense, and if only a single value is needed any reference to the multiple can be removed, simplifying the code. I hope these changes now make more sense, thanks for catching them!

Reply:
Yes, that makes sense. Thanks for fixing!